### ISSUE TITLE
movement code

### ISSUE THREAD

**Issue by ghost** [samhogan/Minecraft-Unity3D#4]
in this repo, i may want to modify the movement speed to see if there is any phenomenon at a couple million block out, (far lands 2.0?) where can i find the code to modify that?

**Comment by Sv443** [comment]
I'm not the dev but here's some variables you can change: https://github.com/samhogan/Minecraft-Unity3D/blob/master/Assets/Scripts/PlayerMovement.cs#L8  
As far as I know you have to go to a coordinate greater than 2,147,483,647 to over- / underflow the integer values and make weird stuff happen.

**Comment by ghost** [comment]
yep, i have the code modded in visual studio IDE and dont know how to load it into unity. my little code knowledge is broken.

**Comment by bgirard** [comment]
Just go in the editor and change the position of the player object.

**Comment by DasBlackfur** [comment]
@MangoDevStudio Yes that is also my point finding that line isnt very chalaging but I have almost no experience with loading code into unity
so If anywone could tell me pls do it. @bgirard Probably you know how to do it.

**Comment by DasBlackfur** [comment]
Okay I got it myself and btw at 100000 this happens:
![grafik](https://user-images.githubusercontent.com/64478051/81597171-9cfeb500-93c5-11ea-9488-cbec8eafe734.png)

**Comment by ghost** [comment]
smarter than me :(, but antvenom would be proud

**Comment by DasBlackfur** [comment]
Hmm I found something else wiered…
If you teleport to 100000 this happens. But if you walk to 100000 everything is fine (exept lighting).
At About 50000 smooth-lighting breaks. At 80000 hard-lighting is broken so I had to diable lighting completly.

**Comment by FinnT730** [comment]
You have to change this line: https://github.com/samhogan/Minecraft-Unity3D/blob/master/Assets/Scripts/PlayerMovement.cs#L8
